CONTRIBUTING.md

```diff
@@ -37,6 +37,14 @@ License Agreement before we can accept your pull request: https://cla.puppet.com
 1. Add the puppetlabs repo as an upstream - `git remote add upstream git@github.com:puppetlabs/bolt`
 1. Make a new branch off of master - `git checkout -b mybranchname`
 1. Commit your changes and add a useful commit message, including what specifically you changed and why - `git commit`
+1. Add a release note with a label using `git notes add`. Valid labels are `@feature`, `@bug`, `@deprecation`, and `@removal`. Notes should be formatted like so:
+    ```
```

Comment:
This may be out of scope for this PR, but could we add a template note that gets copied with `-C`? https://git-scm.com/docs/git-notes#Documentation/git-notes.txt--Cltobjectgt